### DISCUSSION TITLE
Find Width of Letters in TextLayer

### DISCUSSION THREAD

**Discussion by bjvesely** [visgl/deck.gl#5877] [General]
How would I find the width of a text label in the TextLayer.  I need this information to draw a border around the letters on the screen. 

Thank  you.

**Comment by Pessimistress** [comment]
See https://github.com/visgl/deck.gl/issues/4579#issuecomment-782544723

**Comment by bjvesely** [comment]
Thank you.  Would I use the getBorderWidth to find the width of text box?

**Reply by Pessimistress** [reply]
`getBorderWidth` renders the border for you. Please refer to the updated documentation.
